src/app/store.ts

```diff
@@ -11,9 +11,4 @@ export const store = configureStore({
 
 export type AppDispatch = typeof store.dispatch;
 export type RootState = ReturnType<typeof store.getState>;
-export type AppThunk<ReturnType = void> = ThunkAction<
-  ReturnType,
-  RootState,
-  unknown,
-  Action<string>
->;
+export type AppThunk<ReturnType = void> = ThunkAction<ReturnType, RootState, unknown, Action<string>>;
```

Comment:
As you said, It would be good to introduce prettier.
https://create-react-app.dev/docs/setting-up-your-editor/#formatting-code-automatically